### PR TITLE
[CELEBORN-266] Fix wrong old version configurations

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### What changes were proposed in this pull request?
Fix wrong old version configurations

### CODE REVIEW

File: tests/spark-it/src/test/scala/org/apache/celeborn/tests/spark/SkewJoinSuite.scala [hunk 1]
```diff
@@ -41,8 +41,8 @@ class SkewJoinSuite extends AnyFunSuite
 
   private def enableRss(conf: SparkConf) = {
     conf.set("spark.shuffle.manager", "org.apache.spark.shuffle.celeborn.RssShuffleManager")
-      .set("spark.rss.master.address", masterInfo._1.rpcEnv.address.toString)
-      .set("spark.rss.shuffle.split.threshold", "10MB")
+      .set("spark.celeborn.master.endpoints", masterInfo._1.rpcEnv.address.toString)
+      .set("celeborn.shuffle.partitionSplit.threshold", "10MB")
```
Comment (pan3793) [inline tests/spark-it/src/test/scala/org/apache/celeborn/tests/spark/SkewJoinSuite.scala:45]: `spark.` prefix

File: common/src/main/scala/org/apache/celeborn/common/util/Utils.scala [hunk 2]
```diff
@@ -499,9 +499,6 @@ object Utils extends Logging {
     val numThreads = defaultNumThreads(numUsableCores)
     conf.setIfMissing(s"celeborn.$module.io.clientThreads", numThreads.toString)
     conf.setIfMissing(s"celeborn.$module.io.clientThreads", numThreads.toString)
-    // TODO remove after releasing 0.2.0
-    conf.setIfMissing(s"rss.$module.io.serverThreads", numThreads.toString)
-    conf.setIfMissing(s"rss.$module.io.clientThreads", numThreads.toString)
```
Comment (pan3793) [inline common/src/main/scala/org/apache/celeborn/common/util/Utils.scala:504]: is this PR for 0.2? if yes, we should keep this for compatibility

Comment (AngersZhuuuu) [reply common/src/main/scala/org/apache/celeborn/common/util/Utils.scala]: ok

File: tests/spark-it/src/test/scala/org/apache/celeborn/tests/spark/ShuffleFallbackSuite.scala [hunk 3]
```diff
@@ -41,8 +41,7 @@ class ShuffleFallbackSuite extends AnyFunSuite
 
   private def enableRss(conf: SparkConf) = {
     conf.set("spark.shuffle.manager", "org.apache.spark.shuffle.celeborn.RssShuffleManager")
-      .set("spark.rss.master.address", masterInfo._1.rpcEnv.address.toString)
-      .set("spark.rss.shuffle.split.threshold", "10MB")
```
Comment (AngersZhuuuu) [inline tests/spark-it/src/test/scala/org/apache/celeborn/tests/spark/ShuffleFallbackSuite.scala:45]: Not need for this UT

**Review by FMX — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**